app-arch/par2cmdline-turbo/par2cmdline-turbo-1.1.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/animetosho/par2cmdline-turbo"
+else
+	SRC_URI="https://github.com/animetosho/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="par2cmdline × ParPar: speed focused par2cmdline fork"
+HOMEPAGE="https://github.com/animetosho/par2cmdline-turbo"
+
+LICENSE="GPL-2"
```

Comment:
should be `GPL-2+`, see: https://github.com/animetosho/par2cmdline-turbo/blob/turbo/src/par2cmdline.cpp#L7C1-L10

Reply:
Thanks for catching that, I've added a commit to update it on the ebuilds for this package.